src/credits/credits.c

```diff
@@ -654,8 +654,8 @@ static void InitCutsceneSpriteResources(u8 idx, CreditsAppWork *work, u8 sprtRes
 
 static void SetPageSysTasks(CreditsAppWork *work) {
     PageWork *ptr = &work->pageWork;
-    ptr->pageDisplayWork.sysTask = sub_0200E33C((UnkSysTaskFunc)TogglePageDisplayCB, &ptr->pageDisplayWork, 0);
-    work->pageWork.scrFlipWork.sysTask = sub_0200E33C((UnkSysTaskFunc)FlipScreensCB, &work->pageWork.scrFlipWork, 1);
+    ptr->pageDisplayWork.sysTask = sub_0200E33C(TogglePageDisplayCB, &ptr->pageDisplayWork, 0);
+    work->pageWork.scrFlipWork.sysTask = sub_0200E33C(FlipScreensCB, &work->pageWork.scrFlipWork, 1);
```

Comment:
phase out work terminology in these fields

Reply:
out of scope for this PR